+bids/+internal/ends_with.m

```diff
@@ -0,0 +1,23 @@
+function res = ends_with(str, pattern)
+  %
+  % Checks id character array 'str' ends with 'pat'
+  %
+  % USAGE  res = bids.internal.endsWith(str, pat)
+  %
+  % str        - character array
+  % pat        - character array
```

Comment:
Will it be a more complexity/performance issue to make it also accept cellstr for pat?

Reply:
Should be easy enough: the idea would be to check if the file ends with any pattern from all the patterns in a cell?

Reply:
FYI: Will most likely move this to a different issue.